internal/command/volumes/create.go

```diff
@@ -42,8 +40,8 @@ func newCreate() *cobra.Command {
 		flag.Int{
 			Name:        "size",
 			Shorthand:   "s",
-			Default:     3,
-			Description: "The size of volume in gigabytes. The default is 3.",
+			Default:     deploy.DefaultVolumeInitialSizeGB,
+			Description: fmt.Sprintf("The size of volume in gigabytes. The default is %d.", deploy.DefaultGPUVolumeInitialSizeGB),
```

Comment:
cobra already shows defaults out of the box, we can remove that part

![image](https://github.com/superfly/flyctl/assets/1713819/f2821541-0a99-46cf-9539-86e591a139eb)